docs/js/custom.js

```
@@ -0,0 +1,140 @@
+document.addEventListener("DOMContentLoaded", function () {
+    document.querySelectorAll("a[href]").forEach((link) => {
+        if (link.hostname !== window.location.hostname) {
+            link.setAttribute("target", "_blank");
+            link.setAttribute("rel", "noopener noreferrer");
+        }
+    });
+});
+
+// © Copyright year. Last updated on Month date, year.
+document.addEventListener("DOMContentLoaded", function () {
+    let today = new Date();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'today' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)